itl/plugins-contrib.d/smart-attributes.conf

```diff
@@ -8,15 +8,46 @@ object CheckCommand "smart-attributes" {
 	command = [ PluginContribDir + "/check_smart_attributes" ]
 
 	arguments = {
-		"-dbj" = {
+		"--path" = {
+			value = "$smart_attributes_smartclt_path$"
+			description = "Specify the path at which the smartctl binary can be found."
+		}
+		"--device" = {
+			value = "$smart_attributes_device$"
+			description = "Insert the device name (e.g. /dev/sda) to monitor"
+		}
+		"--regex" = {
```

Comment:
This option is not available in the last release `v1.6`, but only on the `master` branch.

Since the last tagged release is six years old, I would suggest adding this parameter anyway, but with a note in the `description` like "(version > 1.6)".

Reply:
Thanks.